Nameserver/src/main/java/be/ua/fti/ei/NextPrevious.java

```diff
@@ -0,0 +1,39 @@
+package be.ua.fti.ei;
+
+/**
+ * Created by asif on 03/05/2021
+ */
+public class NextPrevious {
+
+    private Integer previous,next,numberOfNodes;
```

Comment:
number of nodes is goed, maar wij waren van plan dat de niet de node een unicast doet naar alle andere nodes maar dat de nameserver dit doet

Reply:
![image](https://user-images.githubusercontent.com/37347211/116903402-aa4b2a00-ac3c-11eb-989e-764e70bee416.png)

Deze was ik van plan om vanuit Naming server te sturen ( = Naming server unicast naar alle nodes) 

zie foto 4 c